Changes

```diff
@@ -2,6 +2,8 @@ Change history for HTML-Form
 
 {{$NEXT}}
     - Perl::Tidy-ed the entire codebase (GH#42) (Julien Fiegehenn)
+    - Split extra packages inside HTML::Form into their own module files
+      and move the documentation into a POD file (GH#43) (Julien Fiegehenn)
```

Comment:
I think this needs updating as well.

Reply:
I'm sorry, I thought I had pushed this, but I had not.